src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -51,18 +56,31 @@ public final void publishWithDelay(final Message message, final long delayMillis
         }
 
         if (config.getDelayType() == DelayType.TTL) {
-            publishChannel.basicPublish(ttlExchange(config),
-                    queueName,
```

Comment:
this should be shardedQueueName if sharding is done. Check if this is an issue with existing code

Reply:
Added sharded queue name